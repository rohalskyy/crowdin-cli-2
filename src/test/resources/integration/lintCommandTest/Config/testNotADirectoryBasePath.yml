"project_id" : "1"
"api_token" : "a"
"base_path" : "/home/file.txt"
"base_url": ""

"preserve_hierarchy": true

files: [
    {
        "source" : "source.md",
        "translation" : "%two_letters_code%.md",
    }
]
