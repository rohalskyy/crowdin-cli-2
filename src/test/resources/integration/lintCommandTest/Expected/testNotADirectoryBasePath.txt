Configuration file is invalid. Check the following parameters in your configuration file:
	- base path is wrong
